### DISCUSSION TITLE
Populate Child Records on Top

### DISCUSSION THREAD

**Discussion by dksinghnith** [drivardxrm/LookupDropdown.PCF#11] [Q&A]
hey @drivardxrm , This is exactly matching our requirement but is it possible to populate related child lookup records automatically when we select record in parent lookup.  

thanks...

**Comment by drivardxrm** [comment]
Hi @dksinghnith , I'm glad that you find the control useful and thanks for your feedback.

I'm not sure I understand your requirement correctly, can you send me a more detailed example?

As I understand you would want to populate a dependant child lookup (with related lookup filtering enabled) automatically when a parent is selected.

What would be the value to put in the child? should it use the **first value** in the filtered list 

That could be possible if I add a boolean parameter (Ex. Autopopulate with first listmember) something like that

**Reply by dksinghnith** [reply]
Hi @drivardxrm , Thanks for your response. Actually we just want to populate all related records of child lookup on top of form once we select parent record. In simple, user don't want to click one more time on child record to view all related records once they select parent record.  thanks...

**Comment by drivardxrm** [comment]
Hi @dksinghnith ,

Again, I'm not sure I understand the requirement. Can you share a printscreen or something?

You are talking about lookup on top of forms. Are you referring to a Business Process Flow that is visible on the top of the form?

Note that you might be able to fullfill your requirement with regular javascript on the form.

**Comment by dksinghnith** [comment]
Hi @drivardxrm , Please find attached screen shot. Here I have 2 lookups, "Business Area" and "Level 1". Once i select a record in Business Area, Level 1 should show all related records automatically on top without clicking in Level 1.

![Lookup](https://user-images.githubusercontent.com/106736147/172113069-669a032c-c88e-4fb8-9d84-49879fbc2f60.jpg)

**Reply by dksinghnith** [reply]
Hi @drivardxrm , Could you plz suggest any solution for above requirement. Thanks...

**Comment by drivardxrm** [comment]
Hi @dksinghnith ,

I'm sorry I still dont understand exactly what you are trying to achieve.

Have you configured the field and controls correctly for Reelated Record Filtering ? see my blog post : https://itmustbecode.com/related-record-filtering-with-the-lookup-dropdown-pcf/

Also, keep in mind that this project is open-source and done on my free time. I don't have the bandwidth currently to make any changes.

**Reply by dksinghnith** [reply]
Thanks @drivardxrm for all your help. I have configured everything as per steps suggested but only thing we r  trying to achieve is to show Level 1 values without any click..but we will figure it out.. Thanks a lot and sorry to trouble you :)
